pkg/vsphere/vm/vm.go

```diff
@@ -762,30 +762,36 @@ func (vm *VirtualMachine) RemoveSnapshotByRef(ctx context.Context, snapshot *typ
 	return object.NewTask(vm.Vim25(), res.Returnval), nil
 }
 
-// CheckHostCompatibility checks that a host is compatible for VM placement.
-func (vm *VirtualMachine) CheckHostCompatibility(ctx context.Context, host *types.ManagedObjectReference) (*object.Task, error) {
```

Comment:
Is there a corresponding govmomi update or was this function unused at this time?